komga-webui/src/locales/en.json

```diff
@@ -550,33 +549,33 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could access file during analysis",
```

Comment:
```suggestion
    "ERR_1000": "Could not access file during analysis",
```

Reply:
just update your commit since you changed that line yourself?